includes/class-wc-gateway-swedbank-pay-cc.php

```diff
@@ -70,7 +118,8 @@ public function __construct() {
 		$this->id           = 'payex_psp_cc';
 		$this->has_fields   = true;
 		$this->method_title = __( 'Credit Card', WC_Swedbank_Pay::TEXT_DOMAIN );
-		$this->icon         = apply_filters( 'wc_swedbank_pay_cc_icon', plugins_url( '/assets/images/creditcards.png', dirname( __FILE__ ) ) );
+		$this->icon         = apply_filters( 'wc_swedbank_pay_cc_icon',
+			plugins_url( '/assets/images/creditcards.png', dirname( __FILE__ ) ) );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function dirname() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)